linkcheck/logger/html.py

```diff
@@ -265,65 +316,108 @@ def write_result(self, url_data):
             self.write('</td><td class="error">')
             self.write(html.escape(_("Error")))
         if url_data.result:
-            self.write(": "+html.escape(url_data.result))
+            self.write(": " + html.escape(url_data.result))
         self.writeln("</td></tr>")
 
     def write_stats(self):
         """Write check statistic infos."""
         self.writeln('<br/><i>%s</i><br/>' % _("Statistics"))
         if self.stats.number > 0:
-            self.writeln(_(
-              "Content types: %(image)d image, %(text)d text, %(video)d video, "
-              "%(audio)d audio, %(application)d application, %(mail)d mail"
-              " and %(other)d other.") % self.stats.link_types)
+            self.writeln(
+                _(
+                    "Content types: %(image)d image, %(text)d text, %(video)d video, "
+                    "%(audio)d audio, %(application)d application, %(mail)d mail"
+                    " and %(other)d other."
+                )
+                % self.stats.link_types
```

Comment:
Ooh, now this may be where that ContentTypes dict with all 0 values might be used (as a prototype that's copied before incrementing the items, I'd hope).

Reply:
Yep.